src/current/_data/releases.yml

```diff
@@ -5325,3 +5325,28 @@
     docker_arm_limited_access: true
   source: true
   previous_release: v23.2.0-beta.3
+
+# Matt's cloud-first test
+- release_name: v23.1.14
+  major_version: v23.1
+  release_date: '2023-12-25'
+  release_type: Production
+  cloud_only: true
+  cloud_only_message_short: 'Available only to select CockroachDB Dedicated clusters'
+  cloud_only_message: > # Leave two blank lines for a new para. No Liquid.
```

Comment:
Here are the three fields in action. When the Cloud-only phase is over, removing the three fields will reveal the download links.